stream-compositions/events/events/grandcentral/party-event.json

```diff
@@ -0,0 +1,284 @@
+                "type": "string"
+              },
+              "type": {
+                "type": "string",
+                "enum": [
+                  "ALIEN_REGISTRATION_NUMBER",
+                  "CUSTOMER_IDENTIFICATION_NUMBER",
+                  "EMPLOYEE_IDENTIFICATION_NUMBER",
+                  "NATIONAL_IDENTITY_NUMBER",
+                  "TAX_IDENTIFICATION_NUMBER",
+                  "SOCIAL_SECURITY_NUMBER",
+                  "DRIVERS_LICENSE_NUMBER",
+                  "PASSPORT_NUMBER",
+                  "IDENTITY_CARD_NUMBER",
+                  "EMPLOYER_IDENTIFICATION_NUMBER"
+                ]
+              }
+            }
+          }
+        },
+        "personName": {
+          "type": "object",
+          "properties": {
+            "familyName": {
+              "type": "string"
+            },
+            "firstName": {
+              "type": "string"
+            },
+            "middleName": {
+              "type": "string"
+            }
+          }
+        },
+        "phoneAddresses": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "countryCode": {
+                "type": "string"
+              },
+              "countryIsoCode": {
+                "type": "string"
+              },
+              "key": {
+                "type": "string"
+              },
+              "number": {
+                "type": "string"
+              },
+              "primary": {
+                "type": "boolean"
+              },
+              "type": {
+                "type": "string",
+                "enum": [
+                  "MOBILE",
+                  "HOME",
+                  "WORK",
+                  "OTHER"
+                ]
+              }
+            }
+          }
+        },
+        "postalAddresses": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "addressLine": {
+                "type": "string"
+              },
+              "buildingNumber": {
+                "type": "string"
+              },
+              "country": {
+                "type": "string"
+              },
+              "countrySubDivision": {
+                "type": "string"
+              },
+              "department": {
+                "type": "string"
+              },
+              "key": {
+                "type": "string"
+              },
+              "postalCode": {
+                "type": "string"
+              },
+              "primary": {
+                "type": "boolean"
+              },
+              "streetName": {
+                "type": "string"
+              },
+              "subDepartment": {
+                "type": "string"
+              },
+              "townName": {
+                "type": "string"
+              },
+              "type": {
+                "type": "string",
+                "enum": [
```

Comment:
`PERMANENT` vs `TEMPORARY` don't belong in the list... the address is likely to be both `RESIDENTIAL` and `TEMPORARY` or `BUSINESS ` and `PERMANENT`
`PO_BOX` probably same thing: I can have a `PO_BOX` as a `CORRESPONDENCE` address.

Reply:
The EBP doesn't persist any user profile information, it is always originated from an external source, which, assuming this event originates from GC, the EBC persistence of the user profile information will also be GC as in https://backbase.atlassian.net/wiki/spaces/GC/pages/4479123733/user-profile-manager-client-api

This way we can completely ignore this information when listening to this event.

Reply:
Something to address in the discussion on the unified spec then